tools/cem-utils/src/types.d.ts

```diff
@@ -20,6 +20,10 @@ export interface Component extends schema.CustomElementDeclaration {
   };
 }
 
+export interface ComponentWithModule extends Component {
+  module: CustomModule;
```

Comment:
I don't think this is correct. There shouldn't be an array of components in a component.

Reply:
I think the `WebTypeElement` type needs to be updated instead.

Reply:
The addition of the module here is necessary to pass module.path along. But you're right WebTypeElement also needs to be adjusted.

Reply:
`CustomModule` has a property called `declarations` that contains an array of `Components`.